### PR TITLE
doc: clarify difference between fs and kcephfs suite

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Fixes: https://tracker.ceph.com/issues/43220
Fixes: https://tracker.ceph.com/issues/43222

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by joscollin** [comment]
There is one more: `multimds`

**Comment by rishabh-d-dave** [comment]
> There is one more: `multimds`

Nice! Mentioned it too. :)

File: doc/dev/developer_guide/index.rst [hunk 1]
```diff
@@ -937,14 +937,17 @@ all the integration tests, for all the Ceph components.
...
 
 `kcephfs <https://github.com/ceph/ceph/tree/master/qa/suites/kcephfs>`_
-  test the CephFS kernel module
+  test CephFS, specifically, CephFS mounted using kernel
 
 `krbd <https://github.com/ceph/ceph/tree/master/qa/suites/krbd>`_
   test the RBD kernel module
 
+`multimds <https://github.com/ceph/ceph/tree/master/qa/suites/multimds>`_
+  test CephFS, specifically, CephFS with multiple MDSs
```
Comment (varshar16) [inline doc/dev/developer_guide/index.rst]: This part can be removed. Otherwise looks good.
```suggestion
  test CephFS with multiple MDSs
```

Comment (rishabh-d-dave) [reply doc/dev/developer_guide/index.rst]: Fixed. Thanks!

**Comment by rishabh-d-dave** [comment]
@batrick Can you please take a final look and merge this PR if it's ready in your opinion?

**Comment by rishabh-d-dave** [comment]
Thank you all.

**Pull request merged**